RecoHGCal/TICL/plugins/PatternRecognitionbyCLUE3D.cc

```diff
@@ -704,10 +837,35 @@ template <typename TILES>
 void PatternRecognitionbyCLUE3D<TILES>::fillPSetDescription(edm::ParameterSetDescription &iDesc) {
   iDesc.add<int>("algo_verbosity", 0);
   iDesc.add<double>("criticalDensity", 4)->setComment("in GeV");
-  iDesc.add<int>("densitySiblingLayers", 3);
+  iDesc.add<double>("criticalSelfDensity", 0.15 /* roughly 1/(densitySiblingLayers+1) */)
+      ->setComment("Minimum ratio of self_energy/local_density to become a seed.");
+  iDesc.add<int>("densitySiblingLayers", 3)
+      ->setComment(
+          "inclusive, layers to consider while computing local density and searching for nearestHigher higher");
   iDesc.add<double>("densityEtaPhiDistanceSqr", 0.0008);
+  iDesc.add<double>("densityXYDistanceSqr", 3.24 /*6.76*/)
+      ->setComment("in cm, 2.6*2.6, distance on the transverse plane to consider for local density");
```

Comment:
Now the default is 1.8*1.8 cm (and not 2.6*2.6)
However, since the default value is set, there is no reason to have it reported in the comment field (especially if it does not correspond)
And, by the way; "in cm" -> "in cm2"